src/Api.Rest.Dtos/Converter/GenericSearchConditionConverter.cs

```diff
@@ -0,0 +1,120 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss IMT (IZfM Dresden)                   */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2015                             */
```

Comment:
Small thing: please check here and in the other new files if the year is correct for the copyright.